facebook/Makefile

```diff
@@ -178,18 +197,18 @@ run-R: $(CIDS) $(CIDS_EXP)
 	grep "scheduled core" tmp ; \
 	[ "$$?" -eq 1 ]
 
-pipeline: scratch init-qualtrics params.json $(WEIGHTS) run-R post-cids post-experimental-cids post-individual post-individual-raceeth post-done tidy
+pipeline: scratch init-qualtrics params.json $(WEIGHTS) run-R post-cids post-experimental-cids post-individual post-individual-raceeth post-done tidy $(WEEKLY_WEIGHTS)
 	grep $(TODAY) params.json
```

Comment:
will this still succeed if params.json has been configured for weekly weights?

Reply:
hmmmm

Reply:
Addressed in https://github.com/cmu-delphi/covidcast-indicators/pull/1572/commits/432eeab9597e5109039150d0cd2cb64ad9d5eed9 by saving an existing params file before running the weekly weights pipeline and restoring after the pipeline is finished.